.circleci/config.yml

```diff
@@ -55,61 +59,26 @@ jobs:
       - samvera/rubocop
 
       - samvera/install_solr_core:
-          solr_config_path: "spec/internal_test_hyrax/solr/conf"
-
-      - run: bundle exec rails g hyrax:hirmeos:install
+          solr_config_path: 'spec/internal_test_hyrax/solr/config'
 
-      # Setup CodeClimate
-      - run:
-          command: |
-            if [[ $(command -v cc-test-reporter) == "" ]]; then
-              curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-              chmod +x ./cc-test-reporter
-            fi
       - run:
-          command: ./cc-test-reporter before-build
+          name: Install Hyrax Orcid
+          command: bundle exec rails g hyrax:orcid:install
```

Comment:
Why do we need to install Hyrax Orcid?

Reply:
Well spotted. I had left a reference to the other gem i copied this config over from.

Reply:
Fixed now.